### DISCUSSION TITLE
Poke했을때 피드백

### DISCUSSION THREAD

**Discussion by act-lamos** [metamong-friends/metamong-friends.github.io#11] [제안하기]
Poke 보냈을 때 이펙트가 없어서 보냈는지 여부를 잘 모르겠어요

**Comment by shlee303** [comment]
Poke 가 보내졌다는 토스트 메시지를 추가하는 방안을 검토해보겠습니다. 좋은 의견 감사합니다!

**Comment by shlee303** [comment]
@act-lamos 지난 업데이트에서 poke 시 snack 메시지+사운드가 추가됐습니다.
덕분에 개선 할 수 있었습니다, 감사합니다!
